HelpSource/Classes/Pclump.schelp

```diff
@@ -0,0 +1,38 @@
+TITLE:: Pclump
+summary:: Clumps the pattern into groups of size n.
+related:: Classes/SequenceableCollection#-clump
+categories:: Streams-Patterns-Events>Patterns>List
+
+DESCRIPTION::
+Takes the source pattern into groupings whose size is given by integers in the pattern n.
```

Comment:
i think "grouping" is an unnecessarily vague way of saying "array" here. i'm also not sure if "takes" is the right verb; "clumps" or "groups" would be more appropriate IMO. you should also cross-reference Array.clump, maybe by adding ", similar to Array:-clump" to the end of the sentence.

Reply:
Groups the source pattern into arrays whose size is given by n.

Reply:
I added similar to SequenceableCollection.clump as Array.clump inherits from this and doesn't have any documentation defined for it.

Reply:
thanks for that, i forget most of the manipulation methods are on seqcoll